lib/galaxy/tools/search/__init__.py

```diff
@@ -305,6 +309,10 @@ def clean(string):
             add_doc_kwds["stub"] = clean(id_stub)
         else:
             add_doc_kwds["stub"] = to_unicode(id)
+        if tool.inputs:
+            data_tool_params = [value for value in tool.inputs.values() if isinstance(value, DataToolParameter)]
+            inputs = list({ext for param in data_tool_params for ext in param.extensions})
+            add_doc_kwds["inputs"] = to_unicode(" ".join(inputs))
```

Comment:
```suggestion
            input_extensions = set()
            for tool_input in tool.inputs.values():
                if isinstance(tool_input, BaseDataToolParameter):
                    for extension in param.extensions:
                        input_extensions.add(extension)
            add_doc_kwds["inputs"] = to_unicode(" ".join(input_extensions))
```
This will also work for collection inputs. This will fail though for inputs that can accept anything (format="data") and it doesn't work for subclassed datatypes.

Reply:
Made this change to account for collection inputs as well.